contributors/devel/development.md

```diff
@@ -345,10 +345,13 @@ To test Kubernetes, you will need to install a recent version of [etcd](https://
 This script will instruct you to make a change to your `PATH`. To make
 this permanent, add this to your `.bashrc` or login script:
 
+
```

Comment:
(nit)
```suggestion

```

Reply:
thanks,
to be clear, is this just deleting a blank line?